index.bs

```diff
@@ -375,6 +375,8 @@ The {{XRSession}} is extended to contain an associated <dfn for="XRSession">set
 
 The {{XRSession}} is extended to contain an associated <dfn for="XRSession">set of active hit test sources for transient input</dfn> that will be used when computing hit test results for transient input.
 
+The application is considered to make an <dfn>unreasonable amount of requests</dfn> when the amount of currently registered requests or recently made requests is considered too high for a legitimate use of the API. This is an OPTIONAL privacy measure that the user agent MAY take to avoid abuse.
```

Comment:
Maybe s/"amount of currently registered requests"/"total number of active hit test sources"? We also have a way of being precise on how many hit test sources are currently active (size of the set of active hit test sources + size of the set of active hit test sources).

Also, nit: s/amount/number in the definition and elsewhere where it's used.

Reply:
Done.